.vscode/launch.json

```diff
@@ -1,6 +1,7 @@
 // SPDX-FileCopyrightText: Microsoft Corporation
 // SPDX-FileCopyrightText: The PFDL VS Code Extension Contributors
 // SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
With this change the file has two `SPDX-License-Identifier` tags which does not work out! If Microsoft released under MIT you need to keep MIT and can't re-license!